pkg/apis/scheduling/v1alpha1/type.go

```diff
@@ -33,14 +33,25 @@ type Queue struct {
 
 // QueueSpec defines the desired state of Queue
 type QueueSpec struct {
-	// TODO
+	QueuePolicy         QueuePolicy    `json:"queuePolicy,omitempty" protobuf:"bytes,1,opt,name=queuePolicy`
+	Priority            *int32         `json:"priority,omitempty" protobuf:"varint,2,opt,name=priority"`
+	PriorityClassName   string         `json:"priorityClassName,omitempty" protobuf:"bytes,3,opt,name=priorityClassName"`
 }
 
 // QueueStatus defines the observed state of Queue
 type QueueStatus struct {
 	// TODO
 }
 
+// +k8s:openapi-gen=true
+// QueuePolicy describes how to scheduling multi queue
```

Comment:
QueuePolicy defines the queueing policy for the elements in the queue

Reply:
ok